spec/controllers/rents_controller.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+def mock_sign_in(user)
+  allow_any_instance_of(described_class).to receive(:authenticate_user!)
```

Comment:
Not a correction right now but remember moving this to a helper file when you need this for another controller spec file